src/ripple/protocol/Rate.h

```diff
@@ -40,10 +41,10 @@ struct Rate
 
     Rate () = default;
 
+    explicit
     Rate (std::uint32_t rate)
```

Comment:
I'm okay with disallowing default construction. If that becomes awkward in the future we can put in a defaulted default constructor.